Assets/MixedRealityToolkit/Definitions/Utilities/SupportedPlatforms.cs

```diff
@@ -16,6 +16,9 @@ public enum SupportedPlatforms
         LinuxStandalone = 1 << 2,
         WindowsUniversal = 1 << 3,
         WindowsEditor = 1 << 4,
```

Comment:
Do we want to add `MacEditor` and `LinuxEditor` at this time as well?

Reply:
we might indeed

Reply:
added